src/relayclient/GsnTestEnvironment.ts

```diff
@@ -70,6 +72,7 @@ class GsnTestEnvironmentClass {
     })
 
     const relayProvider = new RelayProvider(new Web3.providers.HttpProvider(_host), config)
+    console.error('== startGSN: ready.')
```

Comment:
this is the only important log for a user - when everything is up. others logs should be silenced in normal cases.